src/controllers/auth.js

```diff
@@ -7,9 +7,9 @@ import { sendDataResponse, sendMessageResponse } from '../utils/responses.js'
 export const login = async (req, res) => {
   const { email, password } = req.body
 
-  if (!email) {
-    return sendDataResponse(res, 400, {
-      email: 'Invalid email and/or password provided'
+  if (!email || !password) {
+    return sendDataResponse(res, 401, {
+      email: 'Invalid email or/and password provided'
```

Comment:
We should keep the error messages as they were before `Invalid email and/or password provided`. The reason for this is we don't want potential attackers to know if they've got the correct email address but wrong password.